prometheus-install/README.md

```diff
@@ -0,0 +1,46 @@
+# Integrate Prometheus into Flink cluster in non-openshift cluster
+
+After deploying Flink cluster, you can then deploy Prometheus to monitor the metrics inside job manager and task manager by following steps:
+
+1. Update the prometheus configuration with the ip address of flink pods to scrape from:
+
+   **Linux:**
+   ```
+   sed -i s/JOB_MANAGER/$(kubectl get pods -lapp=recommendation-app -n flink -o=jsonpath="{range .items[*]}{.status.podIP}{','}{end}" | cut -d ',' -f1)/g prometheus-install/prometheus-config.yaml
+   sed -i s/TASK_MANAGER/$(kubectl get pods -lapp=recommendation-app -n flink -o=jsonpath="{range .items[*]}{.status.podIP}{','}{end}" | cut -d ',' -f2)/g prometheus-install/prometheus-config.yaml
+   ```
+   **MacOS**
+   ```
+   sed -i '' s/JOB_MANAGER/$(kubectl get pods -lapp=recommendation-app -n flink -o=jsonpath="{range .items[*]}{.status.podIP}{','}{end}" | cut -d ',' -f1)/g prometheus-install/prometheus-config.yaml
+   sed -i '' s/TASK_MANAGER/$(kubectl get pods -lapp=recommendation-app -n flink -o=jsonpath="{range .items[*]}{.status.podIP}{','}{end}" | cut -d ',' -f2)/g prometheus-install/prometheus-config.yaml
+   ```
+   Note: Here we assume there's only 1 job manager and 1 task manager. If you deployed more than that, please update the `prometheus-config.yaml` file.   
+
+2. Install prometheus, configuration, and service:
+   ```
+   kubectl apply -f prometheus-install -n flink
```

Comment:
would this command apply everything under prometheus-install directory? if yes, I think it might be confusing because this directory has flink-monitor.yaml etc. Maybe we should have a sub directory for non openshift yamls only.

Reply:
Oops, you're right! Let me update it.